packages/example-next/chains.ts

```diff
@@ -153,6 +136,29 @@ export const CHAINS: { [chainId: number]: BasicChainInformation | ExtendedChainI
   },
 }
 
+export const CHAINS: ChainConfig = {
+  ...MAINNET_CHAINS,
+  ...TESTNET_CHAINS,
+  3: {
+    urls: [process.env.infuraKey ? `https://ropsten.infura.io/v3/${process.env.infuraKey}` : ''].filter(
```

Comment:
not important - but an `infuraURLforNetwork` function and a `filterValidUrls` function would make this cleaner imo. lots of repeated code

Reply:
I hope it's better now!